File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int og_length = nums.size();
        vector<int> ans(2 * og_length);
        for(int i = 0; i < og_length; ++i) {
            ans[i] = ans[i + og_length] = nums[i];
        }
        return ans;
    }
        
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = []
        
        for i in range(len(nums)):
            left = 1
            right = 1
            for x in nums[:i]:
                left *= x
            for y in nums[i+1:]:
                right *= y
            res.append((left*right))
        return res
            
            